smarkets/streaming_api/framing.py

```diff
@@ -84,9 +87,9 @@ def uleb128_decode(string):
 
     while True:
         try:
-            current_byte = indexbytes(string, position)
+            current_byte = to_decode[position]
```

Comment:
@AdamStelmaszczyk I'm not 100% sure, but I think you should leave the `indexbytes` thing here instead of indexing.